backend/src/utils/config.loader.mock.js

```diff
@@ -0,0 +1,13 @@
+export function configOptions(
```

Comment:
rename to `createConfigMock`...its a factory functions...good practice to use create in the name if something is created